src/docs/sdk/metrics.mdx

```diff
@@ -43,9 +43,8 @@ exist:
 * **Background Aggregation:** SDKs are encouraged to defer flushing and aggregation
   into a background thread.  For SDKs where `fork()` is to be expected, the background
   thread needs to ensure it restarts after fork (eg: python).
-* **Tagging:** metrics are tagged by key value pairs, where each key can have more
-  than one value.  Both keys and values are limited to strings, but the SDK can
-  expose other types if it has a reasonable way to stringify these.
+* **Tagging:** metrics are tagged by key value pairs. Both keys and values are limited to strings,
```

Comment:
LGTM.

re: David's suggestion, maybe the following:
```suggestion
* **Tagging:** metrics are tagged by unique key value pairs. Both keys and values are limited to strings,
```
but I think it's ok to merge as-is.

Reply:
`(a, 1)` and `(a, 2)` are still unique pairs.